### PR TITLE
chore(ci): clean up workflow names

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

**Review by p5 — DISMISSED** [review]

File: .github/workflows/build-coreos-bluefin.yml
Comment (m2Giles) [inline .github/workflows/build-coreos-bluefin.yml:1]: Unless we aren't doing Aurora as well, we should probably have Bluefin in the title.

Comment (castrojo) [reply .github/workflows/build-coreos-bluefin.yml:1]: Wouldn't it just be one workflow for both?

Comment (m2Giles) [reply .github/workflows/build-coreos-bluefin.yml:1]: Unfortunately no. We have brand_name as an input which is how bluefin or Aurora is decided to be built.

Even as I think about GTS vs Latest, I may have to completely restructure the matrix for the reusable build to handle across versions.

Comment (m2Giles) [reply .github/workflows/build-coreos-bluefin.yml:1]: I could possibly look into some sort of matrix option on our current workflow to add coreOS as a build option

Comment (castrojo) [reply .github/workflows/build-coreos-bluefin.yml:1]: Ok I'll just wait post-coreos kernel and then reapply what makes sense.

**Review by m2Giles — APPROVED** [review]

**Pull request merged**